warning: inlining failed in call to 'ngtcp2_objalloc_strm_get': call is unlikely and code size would grow [-Winline]

These do not break the build, but do not look good still:
```
In function 'conn_recv_stream',
    inlined from 'conn_recv_pkt' at /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_conn.c:9233:12:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_objalloc.h:75:25: warning: inlining failed in call to 'ngtcp2_objalloc_strm_get': call is unlikely and code size would grow [-Winline]
   75 |     inline static TYPE *ngtcp2_objalloc_##NAME##_get(                          \
      |                         ^~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_conn.h:358:1: note: in expansion of macro 'ngtcp2_objalloc_def'
  358 | ngtcp2_objalloc_def(strm, ngtcp2_strm, oplent);
      | ^~~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_conn.c:7150:12: note: called from here
 7150 |     strm = ngtcp2_objalloc_strm_get(&conn->strm_objalloc);
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'conn_recv_stop_sending',
    inlined from 'conn_recv_pkt' at /opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_conn.c:9255:12:
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_net_ngtcp2/ngtcp2/work/ngtcp2-0.17.0/lib/ngtcp2_objalloc.h:75:25: warning: inlining failed in call to 'ngtcp2_objalloc_strm_get': call is unlikely and code size would grow [-Winline]
   75 |     inline static TYPE *ngtcp2_objalloc_##NAME##_get(                          \
      |                         ^~~~~~~~~~~~~~~~
```

--- a/configure.ac
+++ b/configure.ac
@@ -625,6 +625,7 @@ if test "x$werror" != "xno"; then
     AX_CHECK_COMPILE_FLAG([-Wunused-macros], [CFLAGS="$CFLAGS -Wunused-macros"])
     AX_CHECK_COMPILE_FLAG([-Wunused-parameter], [CFLAGS="$CFLAGS -Wunused-parameter"])
     AX_CHECK_COMPILE_FLAG([-Wredundant-decls], [CFLAGS="$CFLAGS -Wredundant-decls"])
+    AX_CHECK_COMPILE_FLAG([-Winline], [CFLAGS="$CFLAGS -Winline"])
     # Only work with Clang for the moment
     AX_CHECK_COMPILE_FLAG([-Wheader-guard], [CFLAGS="$CFLAGS -Wheader-guard"])
     AX_CHECK_COMPILE_FLAG([-Wsometimes-uninitialized], [CFLAGS="$CFLAGS -Wsometimes-uninitialized"])

--- a/lib/ngtcp2_acktr.c
+++ b/lib/ngtcp2_acktr.c
@@ -29,6 +29,8 @@
 #include "ngtcp2_macro.h"
 #include "ngtcp2_tstamp.h"
 
+ngtcp2_objalloc_def(acktr_entry, ngtcp2_acktr_entry, oplent);
+
 static void acktr_entry_init(ngtcp2_acktr_entry *ent, int64_t pkt_num,
                              ngtcp2_tstamp tstamp) {
   ent->pkt_num = pkt_num;

--- a/lib/ngtcp2_acktr.h
+++ b/lib/ngtcp2_acktr.h
@@ -65,7 +65,7 @@ typedef struct ngtcp2_acktr_entry {
   };
 } ngtcp2_acktr_entry;
 
-ngtcp2_objalloc_def(acktr_entry, ngtcp2_acktr_entry, oplent);
+ngtcp2_objalloc_decl(acktr_entry, ngtcp2_acktr_entry, oplent);
 
 /*
  * ngtcp2_acktr_entry_objalloc_new allocates memory for ent, and

--- a/lib/ngtcp2_conn.c
+++ b/lib/ngtcp2_conn.c
@@ -50,6 +50,8 @@
    packet payload that should be coalesced to a long packet. */
 #define NGTCP2_MIN_COALESCED_PAYLOADLEN 128
 
+ngtcp2_objalloc_def(strm, ngtcp2_strm, oplent);
+
 /*
  * conn_local_stream returns nonzero if |stream_id| indicates that it
  * is the stream initiated by local endpoint.

--- a/lib/ngtcp2_conn.h
+++ b/lib/ngtcp2_conn.h
@@ -355,7 +355,7 @@ ngtcp2_static_ringbuf_def(dcid_retired, NGTCP2_MAX_DCID_RETIRED_SIZE,
 ngtcp2_static_ringbuf_def(path_challenge, 4,
                           sizeof(ngtcp2_path_challenge_entry));
 
-ngtcp2_objalloc_def(strm, ngtcp2_strm, oplent);
+ngtcp2_objalloc_decl(strm, ngtcp2_strm, oplent);
 
 struct ngtcp2_conn {
   ngtcp2_objalloc frc_objalloc;

--- a/lib/ngtcp2_ksl.c
+++ b/lib/ngtcp2_ksl.c
@@ -35,6 +35,8 @@
 
 static ngtcp2_ksl_blk null_blk = {{{NULL, NULL, 0, 0, {0}}}};
 
+ngtcp2_objalloc_def(ksl_blk, ngtcp2_ksl_blk, oplent);
+
 static size_t ksl_nodelen(size_t keylen) {
   return (sizeof(ngtcp2_ksl_node) + keylen - sizeof(uint64_t) + 0xfu) &
          ~(uintptr_t)0xfu;

--- a/lib/ngtcp2_ksl.h
+++ b/lib/ngtcp2_ksl.h
@@ -106,7 +106,7 @@ struct ngtcp2_ksl_blk {
   };
 };
 
-ngtcp2_objalloc_def(ksl_blk, ngtcp2_ksl_blk, oplent);
+ngtcp2_objalloc_decl(ksl_blk, ngtcp2_ksl_blk, oplent);
 
 /*
  * ngtcp2_ksl_compar is a function type which returns nonzero if key

--- a/lib/ngtcp2_objalloc.h
+++ b/lib/ngtcp2_objalloc.h
@@ -65,15 +65,25 @@ void ngtcp2_objalloc_free(ngtcp2_objalloc *objalloc);
 void ngtcp2_objalloc_clear(ngtcp2_objalloc *objalloc);
 
 #ifndef NOMEMPOOL
-#  define ngtcp2_objalloc_def(NAME, TYPE, OPLENTFIELD)                         \
+#  define ngtcp2_objalloc_decl(NAME, TYPE, OPLENTFIELD)                        \
     inline static void ngtcp2_objalloc_##NAME##_init(                          \
         ngtcp2_objalloc *objalloc, size_t nmemb, const ngtcp2_mem *mem) {      \
       ngtcp2_objalloc_init(                                                    \
           objalloc, ((sizeof(TYPE) + 0xfu) & ~(uintptr_t)0xfu) * nmemb, mem);  \
     }                                                                          \
                                                                                \
-    inline static TYPE *ngtcp2_objalloc_##NAME##_get(                          \
-        ngtcp2_objalloc *objalloc) {                                           \
+    TYPE *ngtcp2_objalloc_##NAME##_get(ngtcp2_objalloc *objalloc);             \
+                                                                               \
+    TYPE *ngtcp2_objalloc_##NAME##_len_get(ngtcp2_objalloc *objalloc,          \
+                                           size_t len);                        \
+                                                                               \
+    inline static void ngtcp2_objalloc_##NAME##_release(                       \
+        ngtcp2_objalloc *objalloc, TYPE *obj) {                                \
+      ngtcp2_opl_push(&objalloc->opl, &obj->OPLENTFIELD);                      \
+    }
+
+#  define ngtcp2_objalloc_def(NAME, TYPE, OPLENTFIELD)                         \
+    TYPE *ngtcp2_objalloc_##NAME##_get(ngtcp2_objalloc *objalloc) {            \
       ngtcp2_opl_entry *oplent = ngtcp2_opl_pop(&objalloc->opl);               \
       TYPE *obj;                                                               \
       int rv;                                                                  \
@@ -91,8 +101,8 @@ void ngtcp2_objalloc_clear(ngtcp2_objalloc *objalloc);
       return ngtcp2_struct_of(oplent, TYPE, OPLENTFIELD);                      \
     }                                                                          \
                                                                                \
-    inline static TYPE *ngtcp2_objalloc_##NAME##_len_get(                      \
-        ngtcp2_objalloc *objalloc, size_t len) {                               \
+    TYPE *ngtcp2_objalloc_##NAME##_len_get(ngtcp2_objalloc *objalloc,          \
+                                           size_t len) {                       \
       ngtcp2_opl_entry *oplent = ngtcp2_opl_pop(&objalloc->opl);               \
       TYPE *obj;                                                               \
       int rv;                                                                  \
@@ -107,14 +117,9 @@ void ngtcp2_objalloc_clear(ngtcp2_objalloc *objalloc);
       }                                                                        \
                                                                                \
       return ngtcp2_struct_of(oplent, TYPE, OPLENTFIELD);                      \
-    }                                                                          \
-                                                                               \
-    inline static void ngtcp2_objalloc_##NAME##_release(                       \
-        ngtcp2_objalloc *objalloc, TYPE *obj) {                                \
-      ngtcp2_opl_push(&objalloc->opl, &obj->OPLENTFIELD);                      \
     }
 #else /* NOMEMPOOL */
-#  define ngtcp2_objalloc_def(NAME, TYPE, OPLENTFIELD)                         \
+#  define ngtcp2_objalloc_decl(NAME, TYPE, OPLENTFIELD)                        \
     inline static void ngtcp2_objalloc_##NAME##_init(                          \
         ngtcp2_objalloc *objalloc, size_t nmemb, const ngtcp2_mem *mem) {      \
       ngtcp2_objalloc_init(                                                    \
@@ -135,6 +140,8 @@ void ngtcp2_objalloc_clear(ngtcp2_objalloc *objalloc);
         ngtcp2_objalloc *objalloc, TYPE *obj) {                                \
       ngtcp2_mem_free(objalloc->balloc.mem, obj);                              \
     }
+
+#  define ngtcp2_objalloc_def(NAME, TYPE, OPLENTFIELD)
 #endif /* NOMEMPOOL */
 
 #endif /* NGTCP2_OBJALLOC_H */

--- a/lib/ngtcp2_rtb.c
+++ b/lib/ngtcp2_rtb.c
@@ -37,6 +37,10 @@
 #include "ngtcp2_unreachable.h"
 #include "ngtcp2_tstamp.h"
 
+ngtcp2_objalloc_def(frame_chain, ngtcp2_frame_chain, oplent);
+
+ngtcp2_objalloc_def(rtb_entry, ngtcp2_rtb_entry, oplent);
+
 int ngtcp2_frame_chain_new(ngtcp2_frame_chain **pfrc, const ngtcp2_mem *mem) {
   *pfrc = ngtcp2_mem_malloc(mem, sizeof(ngtcp2_frame_chain));
   if (*pfrc == NULL) {

--- a/lib/ngtcp2_rtb.h
+++ b/lib/ngtcp2_rtb.h
@@ -86,7 +86,7 @@ struct ngtcp2_frame_chain {
   };
 };
 
-ngtcp2_objalloc_def(frame_chain, ngtcp2_frame_chain, oplent);
+ngtcp2_objalloc_decl(frame_chain, ngtcp2_frame_chain, oplent);
 
 /*
  * ngtcp2_bind_frame_chains binds two frame chains |a| and |b| using
@@ -271,7 +271,7 @@ struct ngtcp2_rtb_entry {
   };
 };
 
-ngtcp2_objalloc_def(rtb_entry, ngtcp2_rtb_entry, oplent);
+ngtcp2_objalloc_decl(rtb_entry, ngtcp2_rtb_entry, oplent);
 
 /*
  * ngtcp2_rtb_entry_new allocates ngtcp2_rtb_entry object, and assigns

